src/main/java/at/hannibal2/skyhanni/features/misc/customscoreboard/ScoreboardElements.kt

```diff
@@ -0,0 +1,413 @@
+package at.hannibal2.skyhanni.features.misc.customscoreboard
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.BitsAPI
+import at.hannibal2.skyhanni.data.HypixelData
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.data.MaxwellAPI
+import at.hannibal2.skyhanni.data.MayorElection
+import at.hannibal2.skyhanni.data.PartyAPI
+import at.hannibal2.skyhanni.data.QuiverAPI
+import at.hannibal2.skyhanni.data.ScoreboardData
+import at.hannibal2.skyhanni.data.SlayerAPI
+import at.hannibal2.skyhanni.features.misc.customscoreboard.CustomScoreboardUtils.getTitleAndFooterAlignment
+import at.hannibal2.skyhanni.mixins.hooks.replaceString
+import at.hannibal2.skyhanni.test.command.ErrorManager
+import at.hannibal2.skyhanni.utils.LorenzUtils.inDungeons
+import at.hannibal2.skyhanni.utils.LorenzUtils.nextAfter
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.RenderUtils.AlignmentEnum
+import at.hannibal2.skyhanni.utils.StringUtils.firstLetterUppercase
+import at.hannibal2.skyhanni.utils.TimeUnit
+import at.hannibal2.skyhanni.utils.TimeUtils
+import at.hannibal2.skyhanni.utils.TimeUtils.formatted
+import io.github.moulberry.notenoughupdates.util.SkyBlockTime
+import java.util.function.Supplier
+
+private val config get() = SkyHanniMod.feature.gui.customScoreboard
+
+// Stats / Numbers
+var purse = "0"
```

Comment:
Im confused why those values exist here. purse and location exists already as api, the other stuff should get saved in their api classes,  or create ones if they dont exist yet.

Reply:
the issue with purse is, when the purse is over the int limit, and I need to translate it, well

Reply:
also does location have the symbol at the fromt?

Reply:
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/fc0a4222-cae4-4ac1-8669-742746114e01)

idk wdym with location and symbol.

Reply:
the symbol for the location line, like here 
![image](https://github.com/hannibal002/SkyHanni/assets/45315647/a7d7e40a-c938-444d-a9f2-8207251bfa13)